### PR TITLE
Improve Mesh layer description

### PR DESCRIPTION
**Pull request by rosaguilar**
### Description

Goal:

Ticket: #2967 #2908 #2943  #2752

- [x] Backport to LTR documentation is required

### Minimal requirements for merging *(for maintainers)*

- [ ] The description of this PR is coherent with the manual and does not provide wrong information.
- [ ] This PR passes the checks.

### CODE REVIEW

File: source/docs/user_manual/working_with_mesh/mesh_properties.rst [hunk 1]
```diff
@@ -142,9 +142,96 @@ The :guilabel:`Source` tab displays basic information about the selected mesh, i
 Use the button :guilabel:`Assign Extra Dataset to Mesh` to add more groups to the current mesh layer. 
 
 Style
------
+=====
 
-TODO
+By clicking the button Style |symbology|, you activate the :guilabel:`Layer Styling` dialog as shown in the following image:
```
Comment (DelazJ) [inline source/docs/user_manual/working_with_mesh/mesh_properties.rst]: You may want to write 
```
By clicking the |symbology| :guilabel:`Style` button...
```
ie we have the symbol, then the name with a guilabel role (as it's something you read in the GUI - and when it's not shown as name but as tooltip, you instead use :sup:)
Also note that this does not 
 ```
activate the :guilabel:`Layer Styling` dialog as shown in the following image:
```
because the layer styling dialog is another panel opened by pressing F7 and showing some of the "Layer properties" dialog which is being shown. And the symbology button enables the Style tab of the dialog not the dialog since you wouldn't get the button otherwise

File: source/docs/user_manual/working_with_mesh/mesh_properties.rst [hunk 2]
```diff
@@ -160,4 +247,25 @@ TODO
       :width: 1.5em
 .. |setProjection| image:: /static/common/mActionSetProjection.png
    :width: 1.5em
-
+.. |symbology| image:: /static/common/symbology.png
```
Comment (DelazJ) [inline source/docs/user_manual/working_with_mesh/mesh_properties.rst]: Some of the buttons below should not be added as images in the pul request they are already available in https://github.com/qgis/QGIS-Documentation/tree/master/static/common folder. No need to readd them as files but good to reference them here.
Also check https://docs.qgis.org/testing/en/docs/documentation_guidelines/substitutions.html for what already exist

File: source/docs/user_manual/working_with_mesh/mesh_properties.rst [hunk 3]
```diff
@@ -160,4 +247,25 @@ TODO
...
+   :width: 1.5em
+.. |rendering_mesh| image:: img/rendering.png
+   :width: 1.5em
+.. |tbutton| image:: img/tbutton.png
+   :width: 1.5em
+.. |contour_active| image:: img/contours_active.png
+   :width: 1.5em
+.. |contour_inactive| image:: img/contours_inactive.png
+   :width: 2em
+.. |classify| image:: img/classify.png
```
Comment (DelazJ) [inline source/docs/user_manual/working_with_mesh/mesh_properties.rst]: This should not be an icon: it's a button with text so should be referenced simply by 
```
:guilabel:`Classify`
```
Same applies to the added but unused mode_continuous.png file

File: source/docs/user_manual/working_with_mesh/mesh_properties.rst [hunk 4]
```diff
@@ -160,4 +247,25 @@ TODO
...
+   :width: 4.5em
+.. |vector_active| image:: img/vector_active.png
+   :width: 2em
+.. |vector_inactive| image:: img/vector_inactive.png
+   :width: 1.5em
+.. |display_user_grid| image:: img/display_user_grid.png
```
Comment (DelazJ) [inline source/docs/user_manual/working_with_mesh/mesh_properties.rst]: Same as the classify button; it should be referenced as a text, ie
```
|checkbox| :guilabel:`Display Vectors on User Grid`
```
Remember to reference the |checkbox| here

File: source/docs/user_manual/working_with_mesh/mesh_properties.rst [hunk 5]
```diff
@@ -160,4 +247,25 @@ TODO
...
-
+.. |symbology| image:: /static/common/symbology.png
+   :width: 2em
+.. |signMinus| image:: /static/common/symbologyRemove.png
+   :width: 1.5em
+.. |signPlus| image:: /static/common/symbologyAdd.png
+   :width: 1.5em
+.. |rendering_mesh| image:: img/rendering.png
+   :width: 1.5em
+.. |tbutton| image:: img/tbutton.png
```
Comment (DelazJ) [inline source/docs/user_manual/working_with_mesh/mesh_properties.rst]: already exists as file and referenced as |general| see https://docs.qgis.org/testing/en/docs/documentation_guidelines/substitutions.html#layer-properties

File: source/docs/user_manual/working_with_mesh/mesh_properties.rst [hunk 6]
```diff
@@ -142,9 +142,96 @@ The :guilabel:`Source` tab displays basic information about the selected mesh, i
...
-TODO
+By clicking the button Style |symbology|, you activate the :guilabel:`Layer Styling` dialog as shown in the following image:
+
+
+.. figure:: img/styling_panel.png
+   :align: center
+
+   Mesh Layer Style
+
+In the tab |rendering_mesh|,  QGIS offers two possibilities to display the grid:
```
Comment (DelazJ) [inline source/docs/user_manual/working_with_mesh/mesh_properties.rst]: Could that be possible to add subsections when you switch tabs within the style tab? See eg Vector [labels](https://docs.qgis.org/testing/en/docs/user_manual/working_with_vector/vector_properties.html#labels-properties) or [diagrams ](https://docs.qgis.org/testing/en/docs/user_manual/working_with_vector/vector_properties.html#diagrams-properties). I think that would help readers to follow

File: source/docs/user_manual/working_with_mesh/mesh_properties.rst [hunk 7]
```diff
@@ -142,9 +142,96 @@ The :guilabel:`Source` tab displays basic information about the selected mesh, i
 Use the button :guilabel:`Assign Extra Dataset to Mesh` to add more groups to the current mesh layer. 
 
 Style
------
+=====
```
Comment (DelazJ) [inline source/docs/user_manual/working_with_mesh/mesh_properties.rst]: Should not change since "Style" is at the same level as "Information" or "Source" in the table of contents
```suggestion
-----
```

File: source/docs/user_manual/working_with_mesh/mesh_properties.rst (hunk 2 above)
Comment (rosaguilar) [reply source/docs/user_manual/working_with_mesh/mesh_properties.rst]: Thanks.

File: source/docs/user_manual/working_with_mesh/mesh_properties.rst (hunk 7 above)
Comment (rosaguilar) [reply source/docs/user_manual/working_with_mesh/mesh_properties.rst]: Thanks for all the suggestions.

File: source/docs/user_manual/working_with_mesh/mesh_properties.rst [hunk 8]
```diff
@@ -142,9 +142,96 @@ The :guilabel:`Source` tab displays basic information about the selected mesh, i
...
+*	Triangular mesh rendering that display triangles
+
+The line width and color can be changed in this dialog or deactivate both grid rendering.
+
+The tab |tbutton| presents the following items:
+
+* groups available in the mesh dataset
+* dataset in the selected group(s), for example, if the layer has a temporal dimension
+* metadata if available
+* blending mode available for the selected dataset, for example: normal, lighten,multiply,…(see blending modes :ref:`blend-modes`)
```
Comment (havatv) [inline source/docs/user_manual/working_with_mesh/mesh_properties.rst]:
```suggestion
* blending mode available for the selected dataset, for example: normal, lighten, multiply,…(see blending modes :ref:`blend-modes`)
```

File: source/docs/user_manual/working_with_mesh/mesh_properties.rst [hunk 9]
```diff
@@ -142,9 +142,96 @@ The :guilabel:`Source` tab displays basic information about the selected mesh, i
...
+.. figure:: img/contours_style.png
+   :align: center
+
+   Styling Contours in a Mesh Layer
+
+Use the slide bar or combo box to set the opacity of the current group.
+
+Use :guilabel:`Load` to adjust the min and max value of the current group.
+
+The :guilabel:`Interpolation` list contains three option to render contours: Linear, Discrete and Exact.
```
Comment (havatv) [inline source/docs/user_manual/working_with_mesh/mesh_properties.rst]:
```suggestion
The :guilabel:`Interpolation` list contains three options to render contours: Linear, Discrete and Exact.
```

File: source/docs/user_manual/working_with_mesh/mesh_properties.rst [hunk 10]
```diff
@@ -142,9 +142,96 @@ The :guilabel:`Source` tab displays basic information about the selected mesh, i
...
+
+The :guilabel:`Color ramp`  drop down list lists the color ramp in your QGIS.
+You can list all available ramps, add a new one, edit or save the one you changed.
+The name of the color ramp will be saved in the configuration and in the QML files.
+
+The :guilabel:`Label unit suffix` is a label added after the value in the legend.
+
+By selecting ‘Continuous’ in the classification :guilabel:`Mode`, QGIS creates classes automatically considering
+the :guilabel:`Min` and :guilabel:`Max` values. With ‘Equal interval’, you only need to select the number of classes
+using the combo box  :guilabel:`Classes`  and pressing the button |classify|.
```
Comment (havatv) [inline source/docs/user_manual/working_with_mesh/mesh_properties.rst]:
```suggestion
using the combo box  :guilabel:`Classes`  and press the button |classify|.
```

File: make.bat [hunk 11]
```diff
@@ -0,0 +1,35 @@
+@ECHO OFF
```
Comment (DelazJ) [inline make.bat]: I'm not sure that this file is related to the Mesh issue....
If it's a valid change, may be worth to add it in a separate PR?

File: make.bat [hunk 12]
```diff
@@ -0,0 +1,35 @@
...
+%SPHINXBUILD% >NUL 2>NUL
+if errorlevel 9009 (
+	echo.
+	echo.The 'sphinx-build' command was not found. Make sure you have Sphinx
+	echo.installed, then set the SPHINXBUILD environment variable to point
+	echo.to the full path of the 'sphinx-build' executable. Alternatively you
+	echo.may add the Sphinx directory to PATH.
+	echo.
+	echo.If you don't have Sphinx installed, grab it from
+	echo.http://sphinx-doc.org/
```
Comment (DelazJ) [inline make.bat]:
```suggestion
	echo.https://sphinx-doc.org/
```

File: make.bat (hunk 11 above)
Comment (rosaguilar) [reply make.bat]: No, this is a windows file. No need to be added. Sorry for that

File: source/docs/user_manual/working_with_mesh/mesh_properties.rst [hunk 13]
```diff
@@ -142,9 +142,96 @@ The :guilabel:`Source` tab displays basic information about the selected mesh, i
...
+The slide bar, combo box and arrows allow to explore another dimension of the data, if available.
+As the slide moves, the metadata is presented accordingly. See the figure below as an example.
+The map canvas will display the selected dataset group as well.
+
+.. figure:: img/groups_available.png
+   :align: center
+
+   Dataset in Selected Group(s)
+
+Under groups click on |contour_inactive| to show contours with default visualization parameters. Then,
```
[the remark this answers is not in the document]
Comment (rosaguilar) [reply source/docs/user_manual/working_with_mesh/mesh_properties.rst]: Apparently, there is no tooltip for this

[the remark this answers is not in the document]
Comment (DelazJ) [reply source/docs/user_manual/working_with_mesh/mesh_properties.rst]: Issue report at https://issues.qgis.org/issues/21634
